lib/gamera/modules/visitable.rb

```diff
@@ -0,0 +1,34 @@
+module Gamera
+  module Visitable
+    # Methods to include on pages that are visitable, i.e. have a URL to visit and are then displayed.
+
+    include Capybara::DSL
+
+    # Open the page url in the browser specified in your Capybara configuration
+    #
+    # @param fail_on_redirect [Boolean] Whether to fail if the site redirects to a page that does not match the url_matcher regex
+    # @raise [WrongPageVisited] if the site redirects to URL that doesn't match the url_matcher regex and fail_on_redirect is true
+    def visit(fail_on_redirect = true)
+      super(url)
+      if fail_on_redirect
+        fail WrongPageVisited, "Expected URL '#{url}', got '#{page.current_url}'" unless displayed?
```

Comment:
Should this be `Gamera::WrongPageVisited` for consistency with the `fail` inside `displayed?` ?